runtime/cudaq/platform/default/rest/helpers/oqc/OQCServerHelp.cpp

```diff
@@ -0,0 +1,298 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#include "common/Logger.h"
+#include "common/RestClient.h"
+#include "common/ServerHelper.h"
+#include "cudaq/utils/cudaq_utils.h"
+#include <fstream>
+#include <thread>
+namespace cudaq {
+
+/// @brief The OQCServerHelper class extends the ServerHelper class to handle
+/// interactions with the OQC server for submitting and retrieving quantum
+/// computation jobs.
+class OQCServerHelper : public ServerHelper {
+private:
+  /// @brief RestClient used for HTTP requests.
+
+  /// @brief Helper method to retrieve the value of an environment variable.
+  std::string getEnvVar(const std::string &key) const;
+
+  /// @brief Helper method to check if a key exists in the configuration.
+  bool keyExists(const std::string &key) const;
+
+  std::vector<std::string> getJobID(int n);
```

Comment:
It might be beneficial to add a code comment explaining this function, e.g., it's not immediately clear what `n` means.

Reply:
This function was renamed to `createNTasks` in the latest commit, so I'll mark this as resolved.